magenta/models/music_vae/js/package.json

```diff
@@ -28,8 +30,10 @@
   },
   "dependencies": {
     "@magenta/core": "^0.0.9",
+    "@types/clone": "^0.1.30",
```

Comment:
Do you also need a dependency directly on 'clone'?

Reply:
Done.